cylc/flow/task_events_mgr.py

```diff
@@ -99,7 +99,7 @@ def log_task_job_activity(ctx, suite, point, name, submit_num=None):
         LOG.debug(ctx_str)
 
 
-class TaskEventsManager(object):
+class TaskEventsManager():
```

Comment:
Python 3 class definition syntax: the official docs actually omit the parentheses here (if no explicit inheritance).  Doesn't really matter though.

Reply:
Still interesting to know, had no idea it worked (actually quickly tested in a terminal here :grimacing: )